styles/base.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: "Montserrat", sans-serif;
+  src: url("./Montserrat/Montserrat-VariableFont_wght.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+body {
+  font-family: "Montserrat", sans-serif;
+  scroll-behavior: smooth;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  box-sizing: border-box;
+  padding-left: 75px;
+  padding-right: 75px;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1150px;
```

Comment:
Maksymalną rozdzielczość ustawiamy dla zawartości, ale nie tła. Tło powinno być nadane elementowi o 100% szerokości, aby nie wyglądało to na ucięte - tak jak obecnie.

Zobacz jak to u mnie wygląda:
![image](https://github.com/devmentor-pl/task-html-and-css-basics/assets/55095257/169c1b2f-4298-4187-a79d-ca261a4935da)

PS. Zobacz jeszcze komentrz do pliku CSS